keycloak/authentication_subflow.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"time"
 )
 
 type AuthenticationSubFlow struct {
```

Comment:
xh

Reply:
Hi @Plotomea82 ,
thanks for the review.
But, sorry, not an experienced go dev: What do you mean by this and "joch"? Are these packages by which I should replace "time"?

Reply:
(Just checked: These are features I didn't touch, but where my IDE decided to add this import automatically. Can revert it, if that's the suggestion)

Reply:
This account keeps posting spam into different repos. Assuming this is not a real person and resolving the conversation